### ISSUE TITLE
"Error: No Compatible Build Found" Upon Opening A Meteor @1.0.3.2 Project

### ISSUE THREAD

**Issue by ypinskiy** [meteor/windows-preview#98]
I made a copy of my existing project which currently works fine on my Linux machine and put it on my Windows machine to test the WINDOWS_PREVIEW 0.3.0 version that came out today. After deleting my .meteor/versions file to fix an update error, I was able to change the version of my project. However, running `meteor update` caused me to have the following errors:

    => Errors while initializing project:
    
    While checking for meteorhacks:cluster@1.6.0:
    error: No compatible build found
    
    While checking for meteorhacks:kadira-binary-deps@1.3.1:
    error: No compatible build found

Correction: I changed the version of the project to one that is recognized as an unreleased Meteor checkout, which required me to run `meteor update` to permanently set the version, thus running into the above error.

**Comment by Slava** [comment]
Yup, probably @arunoda would need to publish the windows versions of these packages once we release Meteor 1.1.

**Comment by arunoda** [comment]
Try to install the latest Kadira version. It has the windows support. 

For cluster, we need to wait a bit.

**Comment by arunoda** [comment]
Update cluster as well. I've deployed a new version without binary modules. It should work on windows without any issues.

**Comment by ypinskiy** [comment]
Ah! I had to manually remove the meteorhacks:* projects from the .meteor/packages file, but then I was able to `meteor update` to set the version number for the project and re-add Cluster and Kadira. `meteorhacks:kadira-profiler` was not able to find a compatible build, but I can live without it. Thanks everyone!

On a side note: I assume for my local package that is failing to build I would have to create it again using the Windows Meteor and copy my old code into the new package?
